sensor/kubernetes/listener/listener.go

```diff
@@ -21,7 +22,7 @@ var (
 )
 
 // New returns a new kubernetes listener.
-func New(client client.Interface, configHandler config.Handler, detector detector.Detector, nodeName string, resyncPeriod time.Duration, traceWriter io.Writer) common.SensorComponent {
+func New(client client.Interface, configHandler config.Handler, detector detector.Detector, nodeName string, resyncPeriod time.Duration, traceWriter io.Writer, queue output.Queue) common.SensorComponent {
```

Comment:
I think we can remove the detector also, right?

Reply:
Yes and done.